### ISSUE TITLE
[BUG] Multi-Project Platform -  Major UI / Navigation issue.  Shows incorrect Project / switches project / can't run jobs.

### ISSUE THREAD

**Issue by Arthidon** [mage-ai/mage-ai#5026]
### Mage version

9.70

### Describe the bug

**Video Link Replicating Issue:** https://1drv.ms/v/s!AlYu3S2PXVu9mcximMZPCamEj6LVfw?e=FqHfvG
The project it shows you are in and the content display on the page sometimes is mismatched
- This can happen in any menu / logs / pipelines / edit pipeline
- Somtimes it will show you the correct project and content but running pipelines will not work or UI will not work
- Sometimes you will get errors telling you the block file doesn't exist (because it is looking in the wrong path)
- PIpeline logs not showing because I do not think the pipeline is actually running due to this issue
- not sure if "When running a block while editing a pipeline, output the block messages to the logs" is working or not... can't get pipeline to run without a trigger

Workaround...Maybe? - Run all pipelines with triggers. The other functionalities of mage get severely hindered due to this issue. I cannot see it's own files  properly.

### To reproduce

see video above:

- navigate to different pipeline menus
- try and run blocks in edit pipeline

**Comment by johnson-mage** [comment]
@Arthidon Got it, we do plan to make some changes to the multi project platform so it can run smoother. But thank you for the detailed PR so we can keep track.

**Comment by Arthidon** [comment]
Yes, i may be working in project > X and someone else may be logged in working in project Y. We understand that we both cannot be working in the same project without stepping on each other. So 1 dev per project atm is what we are doing.

**Comment by wangxiaoyou1993** [comment]
Got it. Then we've merged the PR to support the active project per user.
https://github.com/mage-ai/mage-ai/pull/4865
That'll be released in version 0.9.71.

**Comment by Arthidon** [comment]
Do you think this issue was also causing the issue stated in #5049?
